autopush/router/fcm.py

```diff
@@ -32,8 +31,87 @@ def __init__(self, ap_settings, router_conf):
         self.auth = router_conf.get("auth")
         self.metrics = ap_settings.metrics
         self._base_tags = []
+        self.creds = {}
```

Comment:
Similar comment for self.router_ttl -- it's really just a local variable used in the _route method -- so a self.router_ttl is not needed